pjsip/include/pjsip-ua/sip_inv.h

```diff
@@ -810,6 +810,10 @@ PJ_DECL(pj_status_t) pjsip_inv_invite( pjsip_inv_session *inv,
  * Create the initial response message for the incoming INVITE request in
  * rdata with  status code st_code and optional status text st_text. Use
  * #pjsip_inv_answer() to create subsequent response message.
+ * Notes: 
+ * - Upon failure, p_tdata might still get set. To avoid leak,
+ *   it needs to be sent or otherwise decrement the reference count using 
+ *   #pjsip_tx_data_dec_ref().
```

Comment:
- while here, perhaps add all params docs too?
- adding a bit background about why p_tdata is set upon failure sounds nice? For example `When this function returning non-PJ_SUCCESS, it may be caused by an unacceptable INVITE request. In such cases the function will generate an appropriate answer in p_tdata, e.g: when session timer header Session-Expires is too low, the generated answer will include Min-SE header. If the generated answer is not sent, it must be destroyed, i.e: using pjsip_tx_data_dec_ref(), to avoid resource leak`.